app/src/main/kotlin/com/wire/android/ui/home/newconversation/common/CreateGroupOrChannelButtons.kt

```diff
@@ -0,0 +1,105 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
+package com.wire.android.ui.home.newconversation.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.Dp
+import com.wire.android.R
+import com.wire.android.model.ClickBlockParams
+import com.wire.android.ui.common.button.WireButtonState
+import com.wire.android.ui.common.button.WirePrimaryButton
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.theme.WireTheme
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.util.ui.PreviewMultipleThemes
+
+@Composable
+fun CreateGroupOrChannelButtons(
+    onCreateNewGroup: () -> Unit,
+    onCreateNewChannel: () -> Unit,
+    modifier: Modifier = Modifier,
+    elevation: Dp = MaterialTheme.wireDimensions.bottomNavigationShadowElevation,
+) {
+    Surface(
+        color = MaterialTheme.wireColorScheme.background,
+        shadowElevation = elevation
+    ) {
+        Column(
+            verticalArrangement = Arrangement.Center,
+            modifier = modifier
+                .height(dimensions().newConversationButtonsHeight)
+                .padding(horizontal = dimensions().spacing16x)
+        ) {
+            WirePrimaryButton(
+                text = stringResource(R.string.label_create_new_channel),
+                onClick = onCreateNewChannel,
+                state = WireButtonState.Default,
+                leadingIcon = {
+                    Image(
+                        painter = painterResource(id = R.drawable.ic_channel),
+                        contentDescription = null,
+                        modifier = Modifier.padding(end = dimensions().spacing12x),
+                        colorFilter = ColorFilter.tint(colorsScheme().onPrimaryButtonEnabled)
+                    )
+                },
+                clickBlockParams = ClickBlockParams(blockWhenSyncing = true, blockWhenConnecting = true),
+                modifier = Modifier.padding(bottom = dimensions().spacing12x)
+            )
```

Comment:
Just to prepare for the future: There will be a feature flag around this, depending if the team/backend has the feature enabled or not.